packages/eslint-plugin-svelte/src/rules/no-add-event-listener.ts

```diff
@@ -0,0 +1,74 @@
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { createRule } from '../utils/index.js';
+import type { SuggestionReportDescriptor } from '../types.js';
+
+export default createRule('no-add-event-listener', {
+	meta: {
+		docs: {
+			description: 'Warns against the use of `addEventListener`',
+			category: 'Best Practices',
+			recommended: false,
+			default: 'warn'
+		},
+		hasSuggestions: true,
+		schema: [],
+		messages: {
+			unexpected:
+				'Do not use `addEventListener`. Use the `on` function from `svelte/events` instead.'
+		},
+		type: 'suggestion',
+		conditions: [
+			{
+				svelteVersions: ['5']
+			}
+		]
+	},
+	create(context) {
+		return {
+			CallExpression(node: TSESTree.CallExpression) {
+				const { callee, arguments: args } = node;
+				let target: string | null = null;
+
+				if (args.length < 2 || args.length > 3) {
```

Comment:
Can we check such weird pattern? 😅

```svelte
<script>
	const foo = ['resize', () => { console.log('resized!') }]
	window.addEventListner(...foo);
</script>
```

Reply:
Even if suggestions aren’t possible, maybe it’s fine to just report any use of `addEventListener`.

Reply:
you're probably right and we don't need to care how many params there are, or what they are

i'll add a test for spread, to make sure we don't provide suggestions for it